common-utils/common-UI/common-UI.js

```diff
@@ -3,6 +3,17 @@ export class CommonUI {
     this.testRunner = inputTestRunner;
   }
 
+  /**
+   * Generate a selector string for a filter based on its field attribute.
+   * CAUTION: retrieving the elements associated with the returned selector string 
+   * may produce multiple filters if they all correspond to the same field attribute 
+   * @param {String} field Field attribute value of the filter
+   * @returns {String}
+   */
+  #generateFilterSearchString(field) {
```

Comment:
is this hash a typo?

Reply:
The hash is to specify the function as a private class method. I made the method private since it does internal work for other public methods and shouldn't necessarily be called in the test cases/from a class instance, but I'm also fine to revert it back to a public method.

Reply:
thanks for explanation, no need to revert, just I am not familiar with this syntax